.github/workflows/changeset-release.yml

```diff
@@ -3,22 +3,22 @@ name: Release
 on:
   push:
     branches:
-      - master
+      - main
 
 jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@master
+        uses: actions/checkout@v3
```

Comment:
those are also a little bit unrelated, but we should at least stick to a specific major version of those actions - so I've replaced those master references here as well